src/perf/lib/RpsClient.cpp

```diff
@@ -397,7 +397,9 @@ RpsConnectionContext::StreamCallback(
     case QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE:
         Worker->Client->StreamContextAllocator.Free(StrmContext);
         MsQuic->StreamClose(StreamHandle);
-        Worker->QueueSendRequest();
+        if (CxPlatTimeMs32() < (Worker->Client->RunTime + 3000)) {
```

Comment:
This is wrong. There should be a diff between times here. Also 3000 ms is too much IMO. 100 ms might be enough.